places-solution-integrations/analytics/attach-data.md

```diff
@@ -0,0 +1,70 @@
+# Attaching Places Data to Analytics Events via Launch Rules
+
+This document assumes that you have Adobe Places implemented in your application. If you need help implementing Adobe Places, see [Places Extensions for AEP SDK](../../configure-places-in-the-sdk/README.md).
+
+Once Places is sending in events for entries and exits, you can leverage Rules in Launch to attach your Places data to all Adobe Analytics events. With your desired property selected in Launch, you can create this type of rule by completing four steps:
+
+- [Create a new `Rule`](#create-a-rule)
+- [Select the `Event` you wish to trigger the rule](#select-an-event)
+- [Add any other `Conditions` required to trigger the action (optional)](#add-conditions)
+- [Select the `Action` to Attach Data and define your payload](#define-the-action)
+
+## Create a Rule
+
+In the `Rules` tab, click the `Create New Rule` button.
```

Comment:
This should be a numbered step and reword as follows:
"1. On the *Rules* tab, click *Create New Rule*.

Reply:
@rekrishn do you mean i should change the style to use **bold** instead of `monospaced`?